### ISSUE TITLE
prompt.sh character

### ISSUE THREAD

**Issue by Tazeg** [arialdomartini/oh-my-git#76]
The character  appears twice in prompt.sh but has not effect if we don't use fonts.
Maybe you could replace it with a symbol, i mean for example like "${omg_has_deletions_symbol:=''}".
So that it could be customed in a theme.

**Comment by pdostal** [comment]
Agree, it will be nice to be able to customize everything from `~/.bashrc` :+1:
For example I must have my own :octocat: fork because of background colors  :frowning: 
![screenshot 2015-05-16 21 11 32](https://cloud.githubusercontent.com/assets/1254493/7667642/2e0915de-fc10-11e4-936a-fab75c2db100.png)

**Comment by Sgiath** [comment]
Yeah but I think better would be file `omg.conf` or something like that where user could set his own settings. For example prompt, symbols, etc. I know it is possible now by editing `prompt.sh` but it is some kind of "hacking" :smile: it would be nice to have "official" way.

**Comment by pdostal** [comment]
It would be possible to create `omg.conf` or better `omg.sh` but it's OK to just store it in `.bashrc` file.
The problem is that we need more properties to customize everything - every **symbol** and every **color**.

**Comment by tomasky** [comment]
All the same.
